src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+├── public/
+│   ├── robots.txt
+│   ├── favicon.svg
+│   └-─ social-image.png
+├── astro.config.mjs
+└── package.json
+
+```
+
+### `src/`
+
+Le dossier src est où se trouve le code source de votre projet. Il comprend :
+
+- [Des composants Pages](/fr/core-concepts/astro-pages/)
+- [Des composants Layouts](/fr/core-concepts/layouts/)
+- [Des composants Astro](/fr/core-concepts/astro-components/)
+- [Des composants Front-end (React, etc.)](/fr/core-concepts/framework-components/)
+- [Des fichiers de style (CSS, Sass)](/fr/guides/styling/)
+- [Du Markdown](/fr/guides/markdown-content/)
+
+Astro traite, optimise et regroupe les fichiers `src/` pour créer le site web final qui est délivré au navigateur. Contrairement au répertoire statique `public/`, les fichiers `src/` sont traités et assemblés pour vous par Astro.
+
+Quelques fichiers (comme les composants Astro) ne sont pas envoyés au navigateur tel quel, mais plutôt en temps qu'HTML statique. D'autres fichiers (comme les CSS) sont envoyés au navigateur, mais peuvent être optimisés ou regroupés avec d'autres fichiers CSS pour améliorer les performances.
+
+### `src/components`
+
+**Les composants** sont généralement du code réutilisables pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de grouper et d'organiser tous les composants de votre projet dans ce dossier.
+
+C'est une convention commune dans les projets Astro, mais c'est pas obligatoire. Organisez vos composants comme vous le voulez, si vous le souhaitez !
+
+### `src/layouts`
+
+[Les composants Layouts](/fr/core-concepts/layouts/) sont des types de composants particuliers qui s'appliquent à des pages. Ils sont généralement utilisés par les [Pages Astro](/fr/core-concepts/astro-pages/) et les [Pages Markdown](/fr/guides/markdown-content/) pour définir leur mise en page.
+
+Comme `src/components`, organisez vos composants Layouts comme vous le souhaitez, ils ne sont pas obligatoires.
+
+### `src/pages`
+
+[Les composants Pages](/fr/core-concepts/astro-pages/) sont des types de composants particuliers utilisés pour créer de nouvelles pages. Une Page peut être un composant Astro (`.astro`) ou un fichier Markdown (`.md`) qui représente une page de contenu pour votre site.
+
+> ⚠️ `src/pages` les un dossier **obligatoire** dans votre projet Astro. Sans ça, votre site n'aura aucune pages ou routes !
+
+### `src/styles`
+
+C'est une convention commune de stocker vos fichiers CSS ou Sass dans le dossier `src/styles`, mais c'est pas obligatoire. Du moment que vos fichiers de style sont quelque pars dans le dossier `src/`, Astro va les gérer et les optimiser.
+
+### `public/`
+
+Le dossier `public/` est là pour les fichiers et les ressources qui ne doivent pas être traités durant le processus de compilation d'Astro. Ces fichiers seront copiés dans le dossier de compilation sans modification.
```

Comment:
"... qui ne doivent pas..." or "... qui n'ont pas besoin..."? My intuition says the first one, but the second one matches the English text better. I'm not sure which one is your goal. (I think the English should change to "...which _shouldn't be_ processed..." or "....which should be copied changed...".)